recipes/_ruby.rb

```diff
@@ -61,11 +62,14 @@
     build_env['CFLAGS']  = '-O0 -g -pipe -mcpu=v9'
     build_env['LDFLAGS'] = '-mcpu=v9'
   end
+
+  patches << 'https://raw.githubusercontent.com/chef/omnibus-software/38e8befd5ecd14b7ad32c4bd3118fe4caf79ee92/config/patches/ruby/ruby-solaris-linux-socket-compat.patch'
```

Comment:
That does pull from master. It's just the canonical URL form of:
https://github.com/chef/omnibus-software/blob/master/config/software/ruby.rb